distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc

```diff
@@ -191,6 +191,11 @@ Search for the existence of items containing multiple words, such as `Text = app
 +
 Search using wildcards, such as `Text = foo*` : Returns items containing words like "food", "fool", etc..
 +
+Wildcards should only be used for single word searches, not for phrases.
```

Comment:
I think we should mention why. Maybe something like "Solr does not support wildcards in search phrases. Wildcards can only be used in searches with single words."

Reply:
Wouldn't that be bleeding out the implementation?

Reply:
Do you mean the whole thing or just my reference to Solr? If you mean Solr, I agree that it should be worded some other way.